server/test/test_scanpy_engine.py

```diff
@@ -78,8 +78,6 @@ def test_obs_and_var_names(self):
     def test_schema(self):
         with open(path.join(path.dirname(__file__), "schema.json")) as fh:
             schema = json.load(fh)
-            print(schema)
```

Comment:
I snuck this in, because it was bugging me and I didn't want to make a whole new PR for removing two print statements.